components/dashboard/src/components/WorkspaceLogs.tsx

```diff
@@ -26,6 +26,7 @@ const lightTheme: ITheme = {
 };
 
 export interface Props {
+    taskId: string;
```

Comment:
Why is the `taskId` relevant if it ends up being unused? Is it some sort of hack to make the logs dependent on it?

Reply:
It's used in one of the debug logs. Should only be part of the debug commit we should remove before merging.